e2e/native-esm/__tests__/native-esm.test.js

```diff
@@ -37,6 +37,13 @@ test('should have correct import.meta', () => {
   expect(
     import.meta.url.endsWith('/e2e/native-esm/__tests__/native-esm.test.js'),
   ).toBe(true);
+  expect(import.meta.resolve('colors'))
+    .endsWith('jest/node_modules/colors/lib/index.js')
```

Comment:
you probably need `slash` or some other way of translating into windows style paths